lib/krb5/pkinit.c

```diff
@@ -1092,18 +1091,6 @@ pk_verify_host(krb5_context context,
     if (ret)
 	return ret;
 
-    if (hi) {
-	ret = hx509_verify_hostname(context->hx509ctx, host->cert,
```

Comment:
A caller to `krb5_init_creds_step()` could specify `hostinfo`, which would no longer be validated by `hx509_verify_hostname()` with this patch?

Reply:
In theory yes, but the only caller is krb5_init_creds_get() that that passes NULL,
and there are no external callers because krb5_init_creds_step was luckily never exported
in a released version.